main.go

```diff
@@ -36,6 +36,20 @@ var gcpProjectNumber = flag.Int64("gcp-project-number", 0,
 	"the gcp project number. If unknown, can be found via 'gcloud projects list'")
 var vpcNetworkName = flag.String("vpc-network-name", "default", "VPC network name")
 
+const (
+	privateSPIFFECertProviderInstanceName = "google_cloud_private_spiffe"
```

Comment:
Any literal that is only used once, please inline it. This indirection just makes things harder to read.

Reply:
Done.